### PR TITLE
Update Get-SmbShare.md

### PR DESCRIPTION
**Pull request by msbemba**
added description for smbshare

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR that addresses a fill in the blank area in the article. Thanks.

**Comment by get-itips** [comment]
better with the example

**Comment by msbemba** [comment]
> Can you please provide more textual details about what the input _is_ so I know how to use it? Perhaps adding an additional example in the Examples section would be helpful.

I have added an example . The only values accepted are CSV and Default.

**Comment by dariomws306** [comment]
I think we should also add possible values here:
image

**Comment by msbemba** [comment]
> I think we should also add possible values here:
> image

Default is a Value, I can replace it with CSV which is another accepted value ..  Let me know.

**Comment by e0i** [comment]
@dariomws306 Any further input?

**Comment by dariomws306** [comment]
Example is ok. But I think we should add possible values to the description. For example:

Specifies the input to this cmdlet. You can use this parameter, or you can pipe the input to this cmdlet. Possible values are:
* Default
* CSV
* SBL
* SR

File: docset/winserver2012r2-ps/smbshare/Get-SmbShare.md [hunk 1]
```diff
@@ -148,6 +148,19 @@ VMS2                          Contoso-FS                    J:\VMS
 
 This example retrieves the SMB shares on the computer that are connect to the SMB server named Contoso-FS.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/smbshare/Get-SmbShare.md]:
```suggestion
This example retrieves the SMB shares on the computer that are connected to the SMB server named Contoso-FS.
```

**Pull request merged**